tests/FSharp.Compiler.Service.Tests/Common.fs

```diff
@@ -19,7 +19,7 @@ open FSharp.Test.Utilities
 
 type Async with
     static member RunImmediate (computation: Async<'T>, ?cancellationToken ) =
-        let cancellationToken = defaultArg cancellationToken Async.DefaultCancellationToken
+        let cancellationToken = defaultArg cancellationToken CancellationToken.None
```

Comment:
I guess doesn't matter really, but just in case - why this change?

Reply:
We should restore this, this is no longer needed, I hope. This was before I tracked down all the code that was calling CancelDefaultToken in test cases, crashing unrelated tests.